pandas/core/generic.py

```diff
@@ -1916,7 +1916,7 @@ def _drop_labels_or_levels(self, keys, axis: int = 0):
         # Perform copy upfront and then use inplace operations below.
         # This ensures that we always perform exactly one copy.
         # ``copy`` and/or ``inplace`` options could be added in the future.
-        dropped = self.copy()
+        dropped = self.copy(deep=False)
```

Comment:
Just trying to be explicit with `copy` instead of relying on signature defaults?

Reply:
this improves perf by avoiding making an actual copy.  safe since this method is private